### PR TITLE
bug fix on setupt.f90

### PR DESCRIPTION
**Pull request by guoqing-noaa**

### CODE REVIEW

File: src/gsi/read_prepbufr.f90 [hunk 1]
```diff
@@ -1918,11 +1918,13 @@ subroutine read_prepbufr(nread,ndata,nodata,infile,obstype,lunout,twindin,sis,&
 ! CSD - temporary hack ( move to prepbufr pre-processing later)
```
Comment (ClaraDraper-NOAA) [inline src/gsi/read_prepbufr.f90:1918]: Could you please expand this comment to note that 187, 181, and 183 are the screen-level obs over land?

File: src/gsi/read_prepbufr.f90 [hunk 2]
```diff
@@ -1918,11 +1918,13 @@ subroutine read_prepbufr(nread,ndata,nodata,infile,obstype,lunout,twindin,sis,&
...
 !             Over-ride QM=9 for sfc obs 
               if (sfctype .and. hofx_2m_sfcfile ) then 
-                if (tob) then!hardwire 187 obs error to 1.2 for now
+                if (tob .and. (kx==187 .or. kx==181 .or. kx==183) ) then!hardwire 181/183/187 obs error
                   tqm(k)=2
-                  obserr(3,k)=1.2
+                  qm=2 ! 2=not checked
+                  if (kx==187) obserr(3,k)=2.2585
```
Comment (ClaraDraper-NOAA) [inline src/gsi/read_prepbufr.f90:1924]: Are the 4 significant figures necessary?

File: src/gsi/read_prepbufr.f90 [hunk 3]
```diff
@@ -1918,11 +1918,13 @@ subroutine read_prepbufr(nread,ndata,nodata,infile,obstype,lunout,twindin,sis,&
 ! CSD - temporary hack ( move to prepbufr pre-processing later)
 !             Over-ride QM=9 for sfc obs 
               if (sfctype .and. hofx_2m_sfcfile ) then 
-                if (tob) then!hardwire 187 obs error to 1.2 for now
+                if (tob .and. (kx==187 .or. kx==181 .or. kx==183) ) then!hardwire 181/183/187 obs error
                   tqm(k)=2
-                  obserr(3,k)=1.2
+                  qm=2 ! 2=not checked
```
Comment (ClaraDraper-NOAA) [inline src/gsi/read_prepbufr.f90:1923]: can you please remove the comment (this was to remind me to check that the QM was changed in the output file).

**Review by ClaraDraper-NOAA — CHANGES_REQUESTED** [review]
Thanks Guoqing - looks good to me. I made some small change requests, to clean up the comments a bit - we can merge it after that.

File: src/gsi/read_prepbufr.f90 (hunk 2 above)
Comment (bigtangram) [reply src/gsi/read_prepbufr.f90:1924]: I don't think so.  I just copied those values from the RAP/HRRR setting. We can fine-tune that later.

Comment (guoqing-noaa) [reply src/gsi/read_prepbufr.f90:1924]: I don't think so. I just copied those values from the RAP/HRRR setting. We can fine-tune that later.

**Review by ClaraDraper-NOAA — APPROVED** [review]

**Pull request merged**